data/hails.txt

```diff
@@ -727,6 +780,7 @@ phrase "friendly civilian"
 		"needs to be hosed down!"
 		"is gonna need one heck of a chemical bath!"
 		"desperately needs a sonic scrubbing!"
+		"is ruined! I knew I should'ved listened to my Magic Conch."
```

Comment:
What to magic conches have to do with a captain's decision to carry smelly cargo? Is this meant to imply that the captain in question may be mentally ill?

Reply:
/shrug

Reply:
Also, "should'ved" is incorrect. It should be "should've".

Reply:
was a typo :) fix it if you want

Reply:
it's your PR though :)